nautobot/extras/tests/example_jobs/test_location_with_custom_field.py

```diff
@@ -11,25 +13,29 @@ class Meta:
         name = "Location and Custom Field Creation"
         description = "Location with a custom field"
 
-    def run(self, data, commit):
-        obj_type = ContentType.objects.get_for_model(Location)
-        cf = CustomField(label="cf1", type=CustomFieldTypeChoices.TYPE_TEXT, default="-")
-        cf.validated_save()
-        cf.content_types.set([obj_type])
+    def run(self):
+        with transaction.atomic():
```

Comment:
Seeing as you can also decorate a method like so:

```python
@transaction.atomic
def run(self):
    # ...
```

We should probably test that works, too.